__tests__/docker-tags.test.ts

```diff
@@ -1,15 +1,20 @@
 import {UPDATER_IMAGE_NAME, PROXY_IMAGE_NAME} from '../src/docker-tags'
+import {getImageName} from '../src/update-containers'
 
 describe('Docker tags', () => {
-  test('UPDATER_IMAGE_NAME', () => {
+  test('UPDATER_IMAGE_NAME uses a pinned version and matches the config Dockerfile', () => {
     expect(UPDATER_IMAGE_NAME).toMatch(
       /^docker\.pkg\.github\.com\/dependabot\/dependabot-updater@sha256:[a-zA-Z0-9]{64}$/
     )
+
+    expect(UPDATER_IMAGE_NAME).toEqual(getImageName('Dockerfile.updater'))
```

Comment:
This is a bit of a belt-and-braces test, but I think it's a useful descriptor of 'image tag _must_ be a SHA' and 'must match the checked in Dockerfile' as it's possible for human error to move away from a SHA to a label without anything breaking right now.